### PR TITLE
[ICP-12169]HEIMAN wall scene keypad:fix automation issue

### PR DESCRIPTION
**Pull request by ZouLianghuiS**
this PR fixed below issues:
1. no device icon in device card
2. no quick control in device plugin
3. no trigger condition in automation

@tpmanley , @greens ,please have a review of my PR, thanks

### CODE REVIEW

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: nit: use tabs and put space after `if`

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: nit: add space after `for`

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: Add `displayed: false` to this event so that it doesn't show up in event history

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: nit: add space after `for`

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: I suggest calling this `childLabel` instead of `componentLabel` since it's not actually be used as the `componentLabel` on the component.

**Review by tpmanley — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (ZouLianghuiS) [reply devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: updated, please have a review again

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: This still needs to be `componentLabel`

File: devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy
Comment (tpmanley) [reply devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy]: Remember to make this change too

**Review by tpmanley — APPROVED** [review]

**Comment by tpmanley** [comment]
Since these are somewhat substantial changes and this PR is targeted directly at `staging` I'm going to wait until after this week's deploy to merge this so that it can have enough time in lower environments to be tested.

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
deploy is done for the week, so I'm merging this now

**Pull request merged**
